benchmark/engine.py

```diff
@@ -83,13 +85,20 @@ def __str__(self):
         raise NotImplementedError()
 
     @classmethod
-    def create(cls, engine_type, gcp_project_id, ibm_model_id, ibm_custom_id):
+    def create(cls, **kwargs):

```

Comment:
you probably want it to be:

def create(engine_type, **kwargs):
...

engine_type is a mandatory parameter

Reply:
My kwargs dictionary includes all the command line arguments. I get an error "create() got multiple values for argument 'engine_type' " if I do create(engine_type, **kwargs). And I enforce that engine_type is a required argument in benchmark.py, so engine_type will always have a non-null value in kwargs.

Reply:
ah got it! that makes sense. But do we need to change the signature of .create? I mean can we do:

from main:

.create(**args)

and keep the signature in engine.py as

.create(engine_type, **kwargs)

if not. don't worry and just ignore this comment